packages/core-v2-typechain/README.md

```diff
@@ -0,0 +1,16 @@
+# Core V2 TypeChain
+Typechain codegen for generating Typescript classes and types for V2 contracts.
+
+## Install
+
+```
+npm i @elementfi/core-v2-typechain
```

Comment:
*yarn

Reply:
This is the package specific README, as it would appear if this package was a standalone NPM package. In that case, they could use npm or yarn (or whatever other package manager). Since npm is the "default", it's pretty common to just show instructions as if that's what you're using and let users alter them as needed if they're using a diff package manager.